TESTS/storage_abstraction/basicAPI/basicAPI.cpp

```diff
@@ -30,10 +30,17 @@
 #include <string.h>
 #include <inttypes.h>
 
+#define ERASE_CASE_TIMEOUT (1000)
+
 using namespace utest::v1;
 
+#if defined(TARGET_K64F)
 extern ARM_DRIVER_STORAGE ARM_Driver_Storage_MTD_K64F;
 ARM_DRIVER_STORAGE *drv = &ARM_Driver_Storage_MTD_K64F;
+#else
+extern ARM_DRIVER_STORAGE ARM_Driver_Storage_(0);
```

Comment:
We should not use `ARM_Driver_Storage_(0)`; it is too generic, and could lead to name-clashes when supporting more than one Storage devices concurrently.